package.json

```diff
@@ -153,6 +153,7 @@
     ],
     "setupFiles": [
       "raf/polyfill",
+      "jest-prop-type-error",
```

Comment:
This will throw errors in test when we get prop-type warnings

Reply:
maybe we can add it as a separate PR? I'm pretty sure this can be merged very quickly.

Reply:
https://github.com/theforeman/foreman/pull/6161

Reply:
Rebased